<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>convert</class>
 <widget class="QDialog" name="convert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pretvornik v deseti≈°ki sistem</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="inNumber"/>
   </item>
   <item>
    <widget class="QLabel" name="lbRez">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
